proposals/lip-0048.md

```diff
@@ -51,16 +40,13 @@ The minimum part of the fee should be burned, this is to avoid that validators c
 We define the following constants:
 
 | Name                       | Type    | Value                                                                 | Description                                                       |
-|----------------------------|---------| ----------------------------------------------------------------------|-------------------------------------------------------------------|
+|----------------------------|---------| -------------------------------------------------------------------|-----------------------------------------------------------|
 | **General Constants**      |         |                                                                       |                                                                   |
-| `MODULE_ID_FEE`            | uint32  | 1                                                                     | Module ID of the Fee module.                                      |
-| **Configurable Constants** |         |                                                                       |                                                                   |
+| `MODULE_NAME_FEE`            | string  | "fee"                                                                     | Module name of the Fee module.                                      |
+| `LENGTH_ADDRESS`    | uint32    |   20    |   The length of an address in bytes.   |
+| **Configurable Constants** |         |  **Mainchain Value**                                                                     |                                                                   |
 | `MIN_FEE_PER_BYTE`         | uint64  | 1000                                                                  | Minimum amount of fee per byte required for transaction validity. |
-| `TOKEN_ID_FEE`             | object  | `TOKEN_ID_LSK` = {<br /> `"chainID": 0`, <br /> `"localID": 0`<br />} | Token ID of the token used to pay the transaction fees.           |
-
-### Extra Fee
-
-The Fee module allows to specify extra fees for each command. This is specified in the module configuration and is written as `extraCommandFee(moduleID, commandID)` in this LIP.  All `(moduleID, commandID)` pairs that do not have a specified extra fee are assumed to have `extraCommandFee(moduleID, commandID) = 0`.
+| `TOKEN_ID_FEE`             | bytes  | `TOKEN_ID_LSK = 0x0000000000000000` | Token ID of the token used to pay the transaction fees.           |
```

Comment:
Since we stopped using the local alias id, the tokenID for LSK is `0x0000000100000000`.